packages/devtools-evm/src/signer/sdk.ts

```diff
@@ -99,10 +97,11 @@ export class GnosisOmniSignerEVM<TSafeConfig extends ConnectSafeConfig> extends
     }
 
     async signAndSendBatch(transactions: OmniTransaction[]): Promise<OmniTransactionResponse> {
-        assert(transactions.length > 0, `signAndSendBatch received 0 transactions`)
+        if (transactions.length === 0) {
```

Comment:
this was added because of a failing test in our test suite 'signAndSendBatch':'should reject with no transactions'. So following TDD, I added this into the code